projects/core/src/config/config.module.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @spartacus-eslint/use-default-provide-config */
```

Comment:
Not sure how the script (if any) works for adding the license. Maybe it checks the very first line in a file and adds the license again if it doesn't find a particular string (this is me just guessing).
For that reason, maybe it's better to specify the rule inline?